### DISCUSSION TITLE
Getting java.io.IOException Failed to bind to 0.0.0.0:0.0.0.0:8500 error when starting an agent controller.

### DISCUSSION THREAD

**Discussion by bhakarswayamavarapu** [Xceptance/XLT#402] [General]
Hi @team,
When I was trying to start the agent controller in the virtual linux server(machine) getting the below error
**java.io.IOException Failed to bind to 0.0.0.0:0.0.0.0:8500** 

I was using the below command:
cd <XLT bin>./agentcontroller.sh to start the agent controller.

Please find attached screenshot for reference:
![MicrosoftTeams-image (3)](https://github.com/Xceptance/XLT/assets/140174174/34776bbe-ce6a-4609-939d-6623e4de8ab8)

Can you please suggest?
Thanks!

**Comment by jowerner** [comment]
This typically happens if the port of the XLT Agent Controller (8500 by default) is already used by another process. It may also be that the Agent Controller is already running (started manually by someone else or automatically when the machine was done booting).

**Reply by rschwietzke** [reply]
The exception also clearly states: "Address already in use"
